os/include/tinyara/lcd/ili9341.h

```diff
@@ -320,3 +322,4 @@ int ili9341_clear(FAR struct lcd_dev_s *dev, uint16_t color);
 #endif
 
 #endif							/* __INCLUDE_TINYARA_LCD_ILI9341_H */
+
```

Comment:
I don't know why you modified this file.

Reply:
yes there was only some coding rule modification in this file, for a uniform format with the latest fb/lcd interface